diff --git a/tools/cbmc/proofs/HTTP/IotHttpsClient_AddHeader/IotHttpsClient_AddHeader_harness.c b/tools/cbmc/proofs/HTTP/IotHttpsClient_AddHeader/IotHttpsClient_AddHeader_harness.c
index 92ba921b5..01f4234dc 100644
--- a/tools/cbmc/proofs/HTTP/IotHttpsClient_AddHeader/IotHttpsClient_AddHeader_harness.c
+++ b/tools/cbmc/proofs/HTTP/IotHttpsClient_AddHeader/IotHttpsClient_AddHeader_harness.c
@@ -43,8 +43,10 @@ void *memcpy(void *dest, const void *src, size_t n) {
 
 void harness() {
   IotHttpsRequestHandle_t reqHandle = allocate_IotRequestHandle();
+/*
   if (reqHandle)
     __CPROVER_assume(is_valid_IotRequestHandle(reqHandle));
+*/
   uint32_t nameLen;
   uint32_t valueLen;
   __CPROVER_assume(nameLen < CBMC_MAX_OBJECT_SIZE);
